src/torchcodec/_core/FFMPEGCommon.h

```diff
@@ -177,4 +177,13 @@ bool canSwsScaleHandleUnalignedData();
 
 void setFFmpegLogLevel();
 
+AVIOContext* AVIOAllocContext(
+    uint8_t* buffer,
+    int buffer_size,
+    int write_flag,
+    void* opaque,
+    int (*read_packet)(void* opaque, uint8_t* buf, int buf_size),
+    int (*write_packet)(void* opaque, const uint8_t* buf, int buf_size),
+    int64_t (*seek)(void* opaque, int64_t offset, int whence));
```

Comment:
Let's move the convenience function type aliases of `AVIO*Function` to here, and then use those in all of our interfaces. That should be easier to grep for, and also easier to tell at a glance that the signatures are the same in different places.

Also, since `AVIOAllocContext` is a function and not a type, I think we should name it `avioAllocContext()`. Otherwise I think readers will assume we're instantiating an object of type `AVIOAllocContext` until they look at the declaration.

Reply:
Yes, correct. We can only use the type aliases when we declare some variable that has that type.